actions/get_formatted_error.yaml

```diff
@@ -0,0 +1,24 @@
+---
+description: "Finds an error in the given task and formats the result into an HTML tagged output"
+enabled: true
+runner_type: "python-script"
+entry_point: get_formatted_error.py
+name: get_formatted_error
+parameters:
+  cmdb_request_item:
+    type: string
+    description: "CMDB request item if the VM was requested through a CMDB"
+    required: false
+  cmdb_request_item_url:
+    type: string
+    description: "CMDB request item url if the VM was requested through a CMDB"
+    required: false
+  sn_update:
```

Comment:
Lets call this something else to know if we need to use newlines vs line breaks. Calling it `sn_update` doesn't really tell the user what it is doing.